src/modules/wallet/web3-react/connectors/TrezorConnector/sendTransactionHandler.ts

```diff
@@ -0,0 +1,59 @@
+import { SupportedChainId } from '@cowprotocol/cow-sdk'
+import { JsonRpcProvider } from '@ethersproject/providers'
+import { serialize } from '@ethersproject/transactions'
+
+import { EthereumTransaction } from '@trezor/connect'
+import { jotaiStore } from 'jotaiStore'
+
+import { gasPriceAtom } from 'modules/gasPirce'
+
+import { getHwAccount } from '../../../api/utils/getHwAccount'
+
+import type { TrezorConnect } from '@trezor/connect-web'
+
+// TODO: use API or Oracle for gas price
+const DEFAULT_GOERLI_GAS_PRICE = 40 * 10 ** 9 // 40 GWEI
```

Comment:
Why don't we have it on Goerli?

Reply:
I don't know why exactly, but there is no value in `GAS_FEE_ENDPOINTS`